### DISCUSSION TITLE
What affect will Amazon's Purchase of iRobot have on the iRobotEducation Division ?

### DISCUSSION THREAD

**Discussion by jerrynsr** [iRobotEducation/create3_docs#148] [Questions] [answered]
Will iRobotEducation get assimilated into the Amazon Collective or will they spin it off to be it's own company.  I hope  iRobotEducation remains intact.

**Comment by shamlian** [comment, marked as answer]
We are unable to discuss details about the planned acquisition, but it’s important to note that the deal has not closed, and iRobot and Amazon continue to operate independently.
